### PR TITLE
docs: Add Workflows basic snippet

### PR DESCRIPTION
**Pull request by camiekim**
## Description

Fixes b/325430814

Intending to use this snippet on the following page: https://cloud.google.com/workflows/docs/create-workflow-terraform

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist

**Readiness**

- [ X] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [ X] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [ X] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [ X] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [X ] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [ X] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [X ] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s):

- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [ ] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [ ] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

File: workflows/basic/main.tf [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+# Create a dedicated service account
+resource "google_service_account" "workflows_service_account" {
+  account_id   = "sample-workflows-sa"
+  display_name = "Sample Workflows Service Account"
+}
+# [END workflows_terraform_serviceaccount_create]
+
+# [START workflows_terraform_workflow_deploy]
+# Create a workflow
+resource "google_workflows_workflow" "workflows_example" {
```
Comment (glasnt) [inline workflows/basic/main.tf:41]: Resources defined once should use `default` name. 
```suggestion
resource "google_workflows_workflow" "default" {
```

File: workflows/basic/main.tf [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+/*
+Create a workflow that reads and stores the current date and then retrieves
+a list of Wikipedia articles related to the day of the week.
+See https://cloud.google.com/workflows/docs/create-workflow-terraform
+before running the code snippet.
+*/
+
+# [START workflows_terraform_api_enable]
+# Enable Workflows API
+resource "google_project_service" "workflows" {
```
Comment (glasnt) [inline workflows/basic/main.tf:25]:
```suggestion
resource "google_project_service" "default" {
```

File: workflows/basic/main.tf [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  service            = "workflows.googleapis.com"
+  disable_on_destroy = false
+}
+# [END workflows_terraform_api_enable]
+
+# [START workflows_terraform_serviceaccount_create]
+# Create a dedicated service account
+resource "google_service_account" "workflows_service_account" {
```
Comment (glasnt) [inline workflows/basic/main.tf:33]:
```suggestion
resource "google_service_account" "default" {
```

**Review by glasnt — APPROVED** [review]
Small suggested edits per styleguide

File: workflows/basic/main.tf [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+}
+# [END workflows_terraform_serviceaccount_create]
+
+# [START workflows_terraform_workflow_deploy]
+# Create a workflow
+resource "google_workflows_workflow" "default" {
+  name            = "sample-workflow"
+  region          = "us-central1"
+  description     = "A sample workflow"
+  service_account = google_service_account.workflows_service_account.id
```
Comment (glasnt) [inline workflows/basic/main.tf:45]:
```suggestion
  service_account = google_service_account.default.id
```

**Comment by camiekim** [comment]
Thanks, @glasnt re: "Resources defined once should use default name"....will do so in the future

**Pull request merged**